src/Sylius/Behat/Client/ResponseChecker.php

```diff
@@ -79,6 +80,9 @@ public function isDeletionSuccessful(Response $response): bool
 
     public function hasAccessDenied(Response $response): bool
     {
+        if (!$response instanceof JWTAuthenticationFailureResponse) {
+            return false;
+        }
```

Comment:
```suggestion
        }

```

Reply:
@lruozzi9 
Still actual, though why it wasn't caught by CS checks in the first place.